templates/emails/transactional/test-email.html

```diff
@@ -3,6 +3,6 @@
 {% block content %}
   {{ render_text("Hi " + name) }}
   {{ render_text("This is a test to make sure that you receive emails sent to your alias <b>" + alias + "</b>.") }}
-  {{ render_text("If you have any questions, feel free to reply to this email.") }}
```

Comment:
I think we can remove this sentence as the support page is already mentioned in the email footer:

<img width="835" alt="image" src="https://user-images.githubusercontent.com/3167780/168320141-c7677f67-9b19-4e1d-8179-5cd888f3e40c.png">

Reply:
Removed it